theseus/src/api/pack/install_mrpack.rs

```diff
@@ -72,6 +72,7 @@ pub async fn install_zipped_mrpack(
 #[theseus_macros::debug_pin]
 pub async fn install_zipped_mrpack_files(
```

Comment:
I should have made this comment a while ago, but I really dislike the api changes we did in the compiler improvements PR. installing a pack from a version ID now takes 3 functions, and it's kind of a mess. is there any way we can get around this?

Reply:
Yeah i dont like it either. 
Basically as far as I can tell the slowdown comes from functions getting re-compiled over and over again with #[tauri::command]

The other obvious option would be to de-modularize it. So one type of version installation would call all its relevant code and dolesnt reuse similar behaviour. This would be less of a mess but very very redundant (possisbly prone to error?) . What do you think?